dependencies.sh

```diff
@@ -4,8 +4,8 @@
 #Final authority on what's required to fully build the project
 
 # byond version
-export BYOND_MAJOR=514
-export BYOND_MINOR=1589
+export BYOND_MAJOR=515
+export BYOND_MINOR=1604
```

Comment:
```suggestion
export BYOND_MINOR=1603
```
Byond 1604 hasn't released yet so that's probably not gonna do much